.github/workflows/render_readme.yml

```diff
@@ -0,0 +1,47 @@
+# Name of the workflow
```

Comment:
Can you pull this file from `packagetemplate` please? The current version doesn't update images.

Reply:
Added in https://github.com/epiverse-trace/datadelay/commit/a68473618a3e77408c56e3f9ec4011975c58f000